types/externs.d.ts

```diff
@@ -118,6 +118,8 @@ export interface Flags extends SharedFlagsSettings {
   configPath?: string;
   /** Run the specified plugins. */
   plugins?: string[];
+  /** If set to true, will skip the initial navigation to about:blank. This option is ignored when using the legacy navigation runner. */
+  skipAboutBlank?: boolean;
```

Comment:
This is the only option in the FR context that isn't present on the legacy flags. @paulirish I remember when we added this you were concerned about exposing it to the user, but I'm not sure how else to do it without creating *another* options object for internal stuff.

Reply:
All good. I'm fine with it moving here

Reply:
if `skipAboutBlank` is going to be exposed, it should just be a setting (and put on `SharedFlagsSettings`). It makes more sense as a config-level setting than a module-level one anyways.

That shouldn't change much about the implementation here, though it'll need a default in `constants.js`, etc. The `internalOptions` stuff could then be deleted, too.